### PR TITLE
Remove Email Connector from repo, ballerina-lang

### PR DESCRIPTION
**Pull request by Maninda**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: distribution/zip/jballerina-tools/LICENSE [hunk 1]
```diff
@@ -23,7 +23,6 @@ ballerina-reflect-0.5.0.jar
 ballerina-mime-1.0.0.jar                                                                            jar            apache2
 ballerina-lang.map-1.1.0-java.jar                                                                   jar            apache2
 ballerina-lang.decimal-1.0.0-java.jar                                                               jar            apache2
-ballerina-email-0.6.0-java.jar                                                                      jar            apache2
```
Comment (BuddhiWathsala) [inline distribution/zip/jballerina-tools/LICENSE:26]: Better not to remove from license file

File: tests/testerina-integration-test/src/test/resources/testng.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@ under the License.
             <class name="org.ballerinalang.testerina.test.negative.SkipTestsTestCase" />
             <class name="org.ballerinalang.testerina.test.negative.InvalidDataProviderTestCase" />
             <class name="org.ballerinalang.testerina.test.DisableTestsTestCase" />
-            <class name="org.ballerinalang.testerina.test.MockTest" />
+<!--            <class name="org.ballerinalang.testerina.test.MockTest" />-->
```
Comment (BuddhiWathsala) [inline tests/testerina-integration-test/src/test/resources/testng.xml:40]: Why we comment this?

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

File: tests/testerina-integration-test/src/test/resources/testng.xml (hunk 2 above)
Comment (Maninda) [reply tests/testerina-integration-test/src/test/resources/testng.xml]: Uncommented them and commented the email relevant tests. Will open issues as discussed with Joe.

File: distribution/zip/jballerina-tools/LICENSE (hunk 1 above)
Comment (Maninda) [reply distribution/zip/jballerina-tools/LICENSE]: Replaced the line.

File: distribution/zip/jballerina-tools/LICENSE [hunk 3]
```diff
@@ -104,7 +104,6 @@ mimepull-1.9.7.jar
 testerina-core-2.0.0-Preview1.jar                                                                   jar            apache2
 netty-handler-4.1.39.Final.jar                                                                      bundle         apache2
 ballerina-java-0.9.0-java.jar                                                                       jar            apache2
-ballerina-email-0.6.0.jar                                                                           jar            apache2
```
Comment (BuddhiWathsala) [inline distribution/zip/jballerina-tools/LICENSE:107]: Another license change

Comment (Maninda) [reply distribution/zip/jballerina-tools/LICENSE]: Replaced

File: tests/testerina-integration-test/src/test/resources/testng.xml (hunk 2 above)
Comment (Maninda) [reply tests/testerina-integration-test/src/test/resources/testng.xml]: The issue created is: https://github.com/ballerina-platform/ballerina-lang/issues/25514

**Review by ldclakmal — APPROVED** [review]
LGTM

**Comment by BuddhiWathsala** [comment]
The reason for commenting out some tests instead of removing those is https://github.com/ballerina-platform/ballerina-lang/issues/25514

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**